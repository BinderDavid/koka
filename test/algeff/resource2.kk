effect heap<s::S> { }

val heap = handler<heap<_s>> { }

fun main() {
  using heap
  42
}
