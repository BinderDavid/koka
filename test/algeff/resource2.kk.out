42
.tag-heap : string
foo : () -> int
heap : forall<a,b,e> (action : () -> <heap<a>|e> b) -> e b
main : () -> console ()